app/javascript/react/components/SessionFilters/SessionFilters.style.tsx

```diff
@@ -21,6 +20,32 @@ import { CloseButton } from "../Map/Legend/Legend.style";
 import { SmallPopup } from "../Popups/Popups.style";
 import { H4, H6 } from "../Typography";
 
+const fadeIn = keyframes`
+  from {
+    opacity: 0;
+    transform: translateY(-1.5rem);
+    max-height: 0;
+  }
+  to {
+    opacity: 1;
+    transform: translateY(0);
+    max-height: 33rem;
+  }
+`;
+
+const fadeOut = keyframes`
+  from {
+    opacity: 1;
+    transform: translateY(0);
+    max-height: 33rem;
+  }
+  to {
+    opacity: 0;
+    transform: translateY(-1.5rem);
+    max-height: 0;
+  }
+`;
+
```

Comment:
The opening animation is smooth, but the closing animation stutters. Can we work on fixing that?